applications/search-api/src/main/java/no/fdk/searchapi/SearchApiApplication.java

```diff
@@ -6,6 +6,8 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
```

Comment:
Hva er bakgrunnen for denne commiten? Hvor er dette swagger documentet ? Er det også en kode-feil ?

Reply:
Hovedmålet var å lese query parametrer dynamisk, så at controller har ikke alle parametrer deklarert som argumenter. Fordi hver gang argumenter var endret, brytet controllertest.
Men, de argumentene hadde litt verdi - nemlig swagger-annotasjoner - og de ville jeg ikke slette.

Så jeg åpnet swagger docs side:
http://localhost:8080/swagger-ui.html

Da fant jeg ut at den pageable parameter ble feilhåndtert fra. Så akkurat denne kommiten fikser annotasjon til pageable (fjerner autogenererte feile paramtrer, og legg til de ekte parametrer)

Jeg tror og håper vi skal snart ha automatiserte tester til api-er, køblet med spesifikasjon. Det mener at vi kan ha feltspesikasjon deklarert suverent måte, og validerte. På den tidspunkt kan vi fjerne alle annotasjoner og slette den hele swagger ting.